package.json

```diff
@@ -19,12 +19,21 @@
   },
   "homepage": "https://github.com/frankielb/frankie-barrios-ux-iop#readme",
   "dependencies": {
-    "eslint-plugin-html": "^5.0.5",
-    "yarn": "^1.16.0"
+    "@banno/polymer": "2.3.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.4.5",
+    "babel-loader": "^8.0.6",
+    "copy-webpack-plugin": "^5.0.3",
+    "css-loader": "^3.0.0",
     "eslint": "^6.0.1",
+    "eslint-plugin-html": "^5.0.5",
+    "file-loader": "^4.0.0",
+    "html-webpack-plugin": "^3.2.0",
     "http-server": "^0.11.1",
```

Comment:
Lets delete this now that you are using WP dev server

Reply:
Are we talking lines 30-32 or just the http-server?

Reply:
@frankielb just the http-server

Reply:
Done